.github/workflows/template-version.yml

```
@@ -0,0 +1,34 @@
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Update autograded template version
+        if: contains(github.event.pull_request.chaned_files, 'templates/autograded')
+        run: |
+          sed -i "s/\"version\": \".*\"/\"version\": \"$(date +"%Y.%m.%d-%H%M")\"/" templates/autograded/version.json
+      - name: Update manual template version
+        if: contains(github.event.pull_request.chaned_files, 'templates/manual')
+        run: |
+          sed -i "s/\"version\": \".*\"/\"version\": \"$(date +"%Y.%m.%d-%H%M")\"/" templates/manual/version.json
+      - name: Commit and push changes
+        run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)